game/domain/map.lua

```diff
@@ -14,7 +14,9 @@ function Map:init(sector_name)
 
   ELEMENT.init(self)
 
-  local specs = DB.loadDomain(sector_name)
+  local specs = SECTORS[sector_name]
```

Comment:
> I suppose calling `DB` explicitly makes it clearer to other people what is happening.

That, and it keeps things decoupled: the code using `DB` doesn't need to know how specs are actually stored inside domains.